backend/src/main/java/io/diveni/backend/controller/WebsocketController.java

```diff
@@ -183,6 +197,28 @@ public synchronized void processVote(@Payload String vote, MemberPrincipal membe
     LOGGER.debug("<-- processVote()");
   }
 
+  @MessageMapping("/vote/autoreveal")
```

Comment:
The endpoints are confusing:
here: `/vote/autoreveal`
in line 153 `/startVoting/autoreveal`

Do not use captial letters in uri. Probably we could make more descriptive endpoints?